composer.json

```diff
@@ -30,78 +30,78 @@
             {
                 "name": "laminas/laminas-developer-tools",
-                "constraint": "^2.0.1",
+                "constraint": "^2.1.1",
                 "prompt": "Would you like to install the developer toolbar?",
                 "module": true,
                 "dev": true
             },
             {
                 "name": "laminas/laminas-cache",
-                "constraint": "^2.9.0",
+                "constraint": "^2.11.1",
                 "prompt": "Would you like to install caching support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-db",
-                "constraint": "^2.10.0",
+                "constraint": "^2.12.0",
                 "prompt": "Would you like to install database support (installs laminas-db)?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-mvc-form",
-                "constraint": "^1.0",
+                "constraint": "^1.1.0",
                 "prompt": "Would you like to install forms support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-json",
-                "constraint": "^3.1.2",
+                "constraint": "^3.2",
                 "prompt": "Would you like to install JSON de/serialization support?"
             },
             {
                 "name": "laminas/laminas-log",
-                "constraint": "^2.11",
+                "constraint": "^2.13.1",
                 "prompt": "Would you like to install logging support?",
                 "module": true
             },
             {
-                "name": "laminas/laminas-mvc-console",
-                "constraint": "^1.2.0",
-                "prompt": "Would you like to install MVC-based console support? (We recommend migrating to zf-console, symfony/console, or Aura.CLI)",
+                "name": "laminas/laminas-cli",
+                "constraint": "^1.1.1",
+                "prompt": "Would you like to install command-line interface support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-mvc-i18n",
-                "constraint": "^1.1.1",
+                "constraint": "^1.2.0",
                 "prompt": "Would you like to install i18n support?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-mvc-plugins",
-                "constraint": "^1.0.1",
+                "constraint": "^1.1.0",
                 "prompt": "Would you like to install the official MVC plugins, including PRG support, identity, and flash messages?",
                 "module": true
             },
             {
                 "name": "laminas/laminas-psr7bridge",
-                "constraint": "^1.2.0",
+                "constraint": "^1.3.1",
                 "prompt": "Would you like to use the PSR-7 middleware dispatcher?"
             },
```

Comment:
Oh, totally forgot. This should be replaced with `laminas/laminas-mvc-middleware` at `^2.0`

Should I do that in another PR?

Reply:
@Xerkus yes, that's ok 👍